### PR TITLE
changed the button size and responsive for the mobile view and laptop…

### LINKED ISSUE #83: Change Ticket View Button Size
### Explanation:

Leave the button the size of the text field, check in which other views the component is used. In the other view you see two buttons and in this one only one and check that it looks good on the different mobile and desktop screens, link attached to Figma for the design guide.

### Screenshots
view_start

### Checklist

- [ ] Check Used Button Component
- [ ] Review the different resolutions that look good.

### References
(https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/Dasher?node-id=407%3A46)

### PR DESCRIPTION
**Pull request by mdrianislam0or1**
… view

### Description
Changed the button size compare with the input field and responsive for the mobile view and laptop view.

Please include a summary of the change.
I use media query to fix the button size with different view.

fixes #83 

#### Screenshots
Mobile view -->
ce1

Laptop view --> 
ce2

Confirm/1 Mobile view--> 
con1

Confirm/1 Laptop view --> 

con2

If applicable, add screenshots about your changes

#### Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How to test it

All codes pushed in the newly created branch fix/#83

If applicable, describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

### Checklist:

- [x] Doesn't break the current code.
- [ ] Passes linters and test, also is building.
- [ ] Doesn't have spelling or grammatical problems.
- [ ] Doesn't have unnecessary comments or debugging code.
- [ ] The branch is updated with the last dev branch changes.

### CODE REVIEW

**Comment by angelik0828** [comment]
I see it much better, but in general the buttons are very thick.
Review design.

image
image
image
image

**Comment by mdrianislam0or1** [comment]
@angelik0828 should I change the button height and border radius again?? #83

File: src/components/Form/Form.styled.jsx [hunk 1]
```diff
@@ -17,6 +17,21 @@ export const FormContainer = styled.form`
...
+  }
+  @media (max-width: 1025px) {
+    width: 70%;
+  }
+  @media (max-width: 500px) {
+    width: 70%;
+  }
+  @media (max-width: 360px) {
+    width: 65%;
+  }
```
Comment (Israel-Laguan) [inline src/components/Form/Form.styled.jsx]: When you use percentage or relative size units, please add lower and upper bound, as a good practice, for example:

`min-width` and `max-width`

@Katsu08  please check good values to set here

File: src/components/FormInput.jsx [hunk 2]
```diff
@@ -48,6 +48,21 @@ export const FormInput = styled.input`
...
+  }
+  @media (max-width: 900px) {
+    width: 55%;
+  }
+  @media (max-width: 500px) {
+    width: 55%;
+  }
+  @media (max-width: 400px) {
+    width: 55%;
+  }
```
Comment (Israel-Laguan) [inline src/components/FormInput.jsx]: When you use percentage or relative size units, please add lower and upper bound, as a good practice, for example:

`min-width` and `max-width`

@Katsu08  please check good values to set here

File: src/components/Form/Form.styled.jsx [hunk 3]
```diff
@@ -60,8 +82,13 @@ export const FormButtonSecondary = styled.button`
...
   @media (min-width: 1025px) {
-    width: 200px;
+    width: 270px;
+  }
+  @media (min-width: 810px) {
+    width: 270px;
+  }
+  @media (min-width: 300px) {
+    width: 270px;
   }
```
Comment (Israel-Laguan) [inline src/components/Form/Form.styled.jsx]: @angelik0828 what breakpoints do you suggest?

**Review by e-azocar — COMMENTED** [review]
Please update your branch with `dev` using 
```
git fetch -a
```
and then
```
git pull origin dev
```

File: src/components/Form/Form.styled.jsx (hunk 1 above)
Comment (heliomar-pena) [reply src/components/Form/Form.styled.jsx]: I think the max-width can be 300px and can put it in "FormContainer"

imagen

And min width 280px? 

imagen

**Comment by Israel-Laguan** [comment]
Please rebase with latest dev when you can, thanks in advance

**Review by heliomar-pena — APPROVED** [review]
Good job! please rebase code

File: src/components/Form/Form.styled.jsx [hunk 4]
```diff
@@ -8,23 +8,28 @@ export const FormContainer = styled.form`
...
+    width: 80%;
+  }
 `
 
 export const ButtonContainer = styled.div`
   display: flex;
   align-items: center;
+  gap: 10px;
+  width: 100%;
+
```
Comment (heliomar-pena) [inline src/components/Form/Form.styled.jsx]: Please remove extra space
```suggestion
```

File: src/components/Form/Form.styled.jsx [hunk 5]
```diff
@@ -8,23 +8,28 @@ export const FormContainer = styled.form`
...
 `
 
 export const FormButtonPrimary = styled.button`
-  width: 169px;
+
```
Comment (heliomar-pena) [inline src/components/Form/Form.styled.jsx]: Please remove extra space

```suggestion
```

Comment (heliomar-pena) [inline src/components/Form/Form.styled.jsx]: Please remove extra space

```suggestion
```

File: src/components/Form/Form.styled.jsx [hunk 6]
```diff
@@ -38,16 +43,15 @@ export const FormButtonPrimary = styled.button`
...
 
-  @media (min-width: 1025px) {
-    width: 200px;
-  }
+  
+  
 `
 export const FormButtonSecondary = styled.button`
-  width: 169px;
+
```
Comment (heliomar-pena) [inline src/components/Form/Form.styled.jsx]: Please remove extra space

```suggestion
```

File: src/components/Form/Form.styled.jsx [hunk 7]
```diff
@@ -60,8 +64,5 @@ export const FormButtonSecondary = styled.button`
     border: solid 1px var(--dark, #000);
     box-shadow: 2px 5px 6px rgb(0 0 0 / 0.3);
   }
-
-  @media (min-width: 1025px) {
-    width: 200px;
-  }
+
```
Comment (heliomar-pena) [inline src/components/Form/Form.styled.jsx]: Please remove extra space

```suggestion
```

File: src/components/FormInput.jsx [hunk 8]
```diff
@@ -48,6 +48,10 @@ export const FormInput = styled.input`
       font-size: 1rem;
     }
   }
+
+ 
+
+
```
Comment (heliomar-pena) [inline src/components/FormInput.jsx]: Please remove extra space

```suggestion
```

**Review by heliomar-pena — APPROVED** [review]
Good work! I left some code-style comments

**Comment by heliomar-pena** [comment]
please try to run eslint

**Review by Israel-Laguan — APPROVED** [review]

File: src/components/Form/Form.styled.jsx (hunk 3 above)
Comment (angelik0828) [reply src/components/Form/Form.styled.jsx]: > @angelik0828 what breakpoints do you suggest?

a maximum of 250 X 50 and a minimum of 200 X 40 can be used for the button, 
taking into account the largest and smallest resolution

**Review by heliomar-pena — APPROVED** [review]
Good work!

**Pull request merged**